app/src/examples/DynamicStylesExample.tsx

```diff
@@ -0,0 +1,186 @@
+import { Text, StyleSheet, View, Button } from 'react-native';
+
+import React from 'react';
+
+import Animated, {
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming,
+} from 'react-native-reanimated';
+
+export default function DynamicStylesExample() {
+  const [extraStyle, setState] = React.useState(false);
+  const [rerender, setRerender] = React.useState(false);
+  const widthShared = useSharedValue(80);
+  const backgroundColorShared = useSharedValue<string>('rgb(255,127,63)');
+  const transformScaleYShared = useSharedValue(1);
+  const animatedStyle = useAnimatedStyle(() => ({ width: widthShared.value }));
+  const animatedStyle2 = useAnimatedStyle(() => {
+    return {
+      backgroundColor: backgroundColorShared.value,
+    };
+  });
+
+  function handleChangeProps() {
+    setState(!extraStyle);
+  }
+  function handleAnimate() {
+    widthShared.value = withTiming(Math.random() * 160);
+    backgroundColorShared.value = withTiming(
+      `rgb(${Math.random() * 256},${Math.random() * 256},${
+        Math.random() * 256
+      })`
+    );
+    transformScaleYShared.value = withTiming(Math.random());
+  }
+
+  function handleForceRerender() {
+    setRerender(!rerender);
+  }
+
+  return (
+    <View style={styles.container}>
+      <Text>State: {String(rerender)}</Text>
+      <View style={styles.row}>
+        <View style={styles.container}>
+          <Text>InlineStyles</Text>
+
+          <Text>Same length array</Text>
+          <Animated.View
+            accessible={rerender}
```

Comment:
Why do we use `accessible` prop here? If you want to force re-render, usually we use `key` prop.

Reply:
Key prop causes the remount of the component.